blueprint/security/system-security-plan.md

```diff
@@ -502,9 +502,9 @@ This section is applicable as the CloudSystem contains Microsoft Teams which pro
 
 #### CloudSystem compliance approach
 
-The CloudSystem inherits the security controls Microsoft have implemented for Microsoft Teams as assessed in the Office 365 IRAP report.
+The CloudSystem inherits the security controls Microsoft have implemented for Microsoft Teams as assessed in the Microsoft 365 IRAP report.
 
-The Annex specifies the controls associated with the self-contained use of Microsoft Teams up to the level of PROTECTED. As per the HybridSystem solution design, agencies have the ability to connect Teams to the Telstra Calling for Office 365 service to allow calling between Teams and traditional telephones. Agencies wishing to use Telstra Calling for Office 365 or another similar services should undertake a security assessment to ensure that the product addresses their security requirements.
+The Annex specifies the controls associated with the self-contained use of Microsoft Teams up to the level of PROTECTED. As per the HybridSystem solution design, agencies have the ability to connect Teams to the Telstra Calling for v 365 service to allow calling between Teams and traditional telephones. Agencies wishing to use Telstra Calling for Microsoft 365 or another similar services should undertake a security assessment to ensure that the product addresses their security requirements.
```

Comment:
Telstra Calling for `v` 365 service

v = Microsoft ?

Reply:
Updated